decentraland/unity-explorer
# feat: added connection retry on realm navigator with realm fallback

## What does this PR change?

Fix #2127 
Adds a retry mechanism for realm change to try 3 times before rolling back to previous realm
...

## How to test the changes?



1. Launch the explorer
2. Teleport to a world and back multiple times
3. Verify it works fine
4. If a teleport to a world fails verify that the rollback to the previous realm is working fine

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
